### DISCUSSION TITLE
No data from the Visible Spectrum Sensor as7262 -- How can you tell if it is a bad unit?

### DISCUSSION THREAD

**Discussion by paulmirel** [STELLA-Landsat/STELLA#19] [Debugging]
I've had four bad units out of a hundred or so. 

How can I tell? All the connections are good (power on V+, good Ground, good connections to i2c SDA and SCK). The unit does not show up on an i2c bus scan, at the address 0x49, like it should. The little green indicator LED blinks about once every second for a short pulse, continuously when power is applied. 

The only solution is to remove and replace the sensor. Doing that on an already built STELLA is difficult. 4 or 5% chance that you may encounter this. 

Others have had this problem with the as7262, and the manufacturer lists it as "Not recommended for new designs", where the as7263 is listed as "In production". 
https://forums.adafruit.com/viewtopic.php?t=199319&start=15

**Comment by rockjock32** [comment]
We've been constructing 10 of STELLA 1.1's for a class and ran into 2 bad vis sensors right off the bat. We got tired of desoldering them from the board so we mocked up a test version using a breadboard as shown here. We found 2 more in our batch of 10 that were bad (40% fail rate!). Thankfully the replacements worked. I'm glad the next version of STELLA will use a different vis sensor. 
![IMG_7738](https://github.com/STELLA-Landsat/STELLA/assets/14187446/9b2e0c62-91da-4b3e-8203-9821c142cef2)
